docs/references/git_configuration.md

```diff
@@ -0,0 +1,59 @@
+---
+sidebar_label: "Git Configuration"
+title: ""
+---
+
+# Git Configuration
+
+Since version **1.10.0**, Epinio supports Git configuration.
```

Comment:
```suggestion
Starting with version **1.10.0**, Epinio supports Git configurations.
```
```suggestion
Epinio supports Git configurations since version **1.10.0**.
```